manala.apt/vars/main.yml

```diff
@@ -165,6 +165,9 @@ manala_apt_repositories_patterns:
   mongodb_3_2:
     source: deb http://repo.mongodb.org/apt/{{ ansible_distribution|lower }} {{ ansible_distribution_release }}/mongodb-org/3.2 main
     key: mongodb
+  mongodb_3_6:
+    source: deb http://repo.mongodb.org/apt/{{ ansible_distribution|lower }} wheezy/mongodb-org/3.6 main
```

Comment:
As far as i can see, mongodb is available for both wheezy and jessie, please, replace `wheezy` by `{{ ansible_distribution_release }}` (same as 3.2)